### PR TITLE
fix(core): chip alignment 

### PR DESCRIPTION
**Pull request by goncalosard**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Build (`yarn build`) was run locally and any changes were pushed
- [ ] Unit tests (`yarn test`) were run locally and passed
- [ ] Visual Regression Tests (`yarn visual-regression`) were run locally and passed
- [ ] Linting (`npm lint`) was run locally and passed

## Pull request type

Please check the type of change your PR introduces:

- [x] Bug fix
- [ ] Feature
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

image

Chips with icon have diferent Y than chips with no icon.

## What is the new behavior?

image

- chip with icon and chip without icon now get the same Y

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Testing

Tested locally

### CODE REVIEW

File: packages/core/src/components/chip/chip.scss [hunk 1]
```diff
@@ -14,7 +14,8 @@
 $border-radius: 100px;
 
 :host {
-  display: inline-flex;
+  display: flex;
```
Comment (danielleroux) [inline packages/core/src/components/chip/chip.scss:17]: Same problem as in #255 we can not remove inline here.

**Review by danielleroux — CHANGES_REQUESTED** [review]

**Comment by goncalosard** [comment]
@danielleroux  Fixed, please review.

File: packages/core/src/components/chip/chip.scss [hunk 2]
```diff
@@ -29,6 +30,11 @@ $border-radius: 100px;
     margin-right: $tiny-space;
   }
 
+  .hidden {
+    width: 0 !important;
+    margin-right: 0 !important;
```
Comment (danielleroux) [inline packages/core/src/components/chip/chip.scss:35]: Is `!important` here necessary?

**Review by nuke-ellington — APPROVED** [review]
lgtm

**Review by danielleroux — APPROVED** [review]
lgtm

**Pull request merged**